Eng 972 improve docs about bundle versions (#152)
entando/entando-docs: vuepress/docs/next/tutorials/ecr/ecr-bundle-versions.md

To each version in the `tags` object must correspond a tag in the git repository provided in the `tarball` field.

## What format should I use to version my bundle?

The ECR support the [semantic versioning 2.0.0](https://semver.org/#semantic-versioning-200), with the possibility to prepend a `v` to the version itself. Some valid bundle versions are:

- 1.0
- v0.1.0-alpha
- 1.0.1-SNAPSHOT
- 22.109.10234-RC.1

## As a bundle developer, how should I create a new version of a bundle?
If you're a bundle developer and you want to release a new version of your bundle, you can simply update the content of your bundle and commit the changes to the bundle git repository. 
Once all the required changes are correct, you can create a new tag in the git repository using the `git tag` command and then publish it to the remote repository. 

For example, let's assume my bundle already has a version `1.0.0` and I want to publish the version `2.0.0` of my bundle, here the commands I need to use to create and publish the new tag:

```
    git tag -a "2.0.0" -m "My new version"
    git push --tags
```

## How to prevent a particular bundle version from being installable?

> If, for some reasons, you don't want a particular version to be available for the installation (anymore), you can proceed like follows:

Author: ```suggestion
If, for some reasons, you don't want a particular version to be available for the install, you can proceed like follows:
```

Reviewer: Small changes on this one. Reword and grammar

```
If, for some reason, you don't want a particular version to be available for installation, you can proceed as follows:

```

1. remove the undesired version from the tag list in your bundle Kubernetes file (`integrity`, `shasum`, `tarball` and `version`)
2. delete your bundle from your cluster using a command like this `kubectl delete -f your-bundle-file.yml [-n your-cluster-namespace]`
3. upload again your bundle to Kubernetes using a command like this `kubectl create -f your-bundle-file.yml [-n your-cluster-namespace]`

> ## My bundle contains a microservice generated with the Entando Component Generator, does the version of the microservice have the same as the bundle version?

Author: ```suggestion
## My bundle contains a microservice generated with the Entando Component Generator, does the version of the microservice have to be the same as the bundle version?
```

> The version of the microservice - or to be more precise the docker image associated with the microservice - are not directly releated to the version of the bundle containing the microservice. 

Author: ```suggestion
The version of the microservice - or to be more precise the docker image associated with the microservice - isn't bound to the version of the bundle containing the microservice itself. 
```
